+vis/grating.m

```diff
@@ -4,12 +4,13 @@
 %  may be windowed by a Gaussian stencil, producing a Gabor patch
 %
 %  Inputs:
-%    t - any signal with which to derive the network ID.  By convetion we
-%      use the t signal
+%    t - The 'time' signal. Used to obtain the *signals* network ID.
```

Comment:
what's with the asterisks around the word 'signals'? I've noticed you've done this in a lot of the documentation but I don't understand what purpose it serves

Reply:
I use it to discriminate the * signals * package from multiple individual signals. I use * as borrowed from the original * signals * readme because that's how Chris used it when doing the same discrimination. I'm fine using Signals instead of * signals * though

Reply:
I don't really care as long as its consistent across all documentation.

Reply:
ok, let's stick with 'Signals' and I'll stop using/get rid of the asterisks